modules/tinymce/src/themes/silver/main/ts/ui/dropdown/CommonDropdown.ts

```diff
@@ -178,7 +178,10 @@ const renderCommonDropdown = <T>(
           onControlDetached(spec, editorOffCell)
         ]),
         AddEventsBehaviour.config(fixWidthBehaviourName, [
-          AlloyEvents.runOnAttached((comp, _se) => spec.listRole === 'listbox' ? Fun.noop : UiUtils.forceInitialSize(comp)),
+          AlloyEvents.runOnAttached((comp, _se) => spec.listRole === 'listbox' ? Fun.noop() : UiUtils.forceInitialSize(comp)),
```

Comment:
I doubt `Fun.noop()` is the correct answer here. If the types want a `void` return, just make it `undefined`?

Reply:
I find `Fun.noop` quite readable here. If it is listbox - no operation. If it is not force initial size. 
I don't like how it looks with undefined, so maybe something like this:
```
if (spec.listRole !== 'listbox') {
  UiUtils.forceInitialSize(comp)
}
```
What do you think?

Reply:
yes, that's also a good way to do it 👍